### PR TITLE
Garnett Immersive cards

### PR DESCRIPTION
**Pull request by SiAdcock**
## What does this change?

Styles for immersive cards

Some bugginess around `z-index` due to the overlay. These bugs will be fixed when I finally re-merge #18470

### CODE REVIEW

File: static/src/stylesheets/module/facia-garnett/item-types/_fc-item--type-immersive.scss [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+        display: block;
+    }
+
+    .fc-item__container {
+        overflow: hidden;
+    }
+
+    .fc-item__content {
+        background-color: $immersive-garnett-main-1;
+        z-index: 1;
```
Comment (SiAdcock) [inline static/src/stylesheets/module/facia-garnett/item-types/_fc-item--type-immersive.scss]: Are we able to remove this `z-index`? Would be great if we didn't need to rely on it!

**Review by SiAdcock — COMMENTED** [review]
👍

**Comment by regiskuckaertz** [comment]
I copied the `FaciaWidths` object because a higher image quality is required for some cards and the only way to play with it is by changing the width. It's a cheap and dirty trick but at least allows us to fiddle with the settings without worrying about side effects, and can be removed eventually.

**Comment by paperboyo** [comment]
> the only way to play with it is by changing the width

Currently, yes. But one day [we should be able to use height](https://github.com/guardian/frontend/issues/12216#issuecomment-196869603), too.

File: common/app/layout/WidthsByBreakpoint.scala [hunk 2]
```diff
@@ -95,6 +95,97 @@ object FaciaWidths {
   }
 }
 
+object GarnettFaciaWidths {
```
Comment (GHaberis) [inline common/app/layout/WidthsByBreakpoint.scala]: Would switch from `FaciaWidths` to the updated `GarnettFaciaWidths ` have any knock on impact elsewhere?

Comment (regiskuckaertz) [reply common/app/layout/WidthsByBreakpoint.scala]: Yes: everywhere cards are used, i.e. fronts, tag pages and paid content fronts, but I'd say this is expected.

**Review by GHaberis — APPROVED** [review]
👍

**Pull request merged**